doc/man/remotectl.xml

```diff
@@ -44,12 +44,12 @@
   </refsynopsisdiv>
 
 
-  <refsect1>
+  <refsect1 id="remotectl-description">
```

Comment:
Ah, makes sense. Could you change the commit message accordingly? Like "Add fixed IDs to avoid unpredictable autogenerated ones"?